### PR TITLE
Remove New Relic JS

### PR DESCRIPTION
**Pull request by dmohs**
Our contract with New Relic ends on June 19.

Did not test.

### CODE REVIEW

**Comment by breilly2** [comment]
I am testing this. I'm seeing an error checking registration: "Cannot read property 'setCustomAttribute' of undefined". Investigating further...

File: src/libs/auth.js [hunk 1]
```diff
@@ -170,17 +170,6 @@ authStore.subscribe(withErrorReporting('Error checking groups for timeout status
...
 }))
 
-/**
- * Developers can get access to a user's ID, so a determined person could compare user IDs to
- * hashes to identify a user in our analytics data. We trust our developers to refrain from
- * doing this.
- */
-authStore.subscribe((state, oldState) => {
-  if (!oldState.registrationStatus && state.registrationStatus) {
-    window.newrelic.setCustomAttribute('userGoogleId', state.user.id)
```
Comment (breilly2) [inline src/libs/auth.js:180]: This was the missing piece.

**Review by breilly2 — COMMENTED** [review]
👍 I tested locally and added an additional commit. I verified that:
* the text "newrelic" does not exist in the code anymore
* no requests from *.newrelic.com or *.nr-data.net show up in the browser network inspector

Since I contributed a commit, maybe someone else should take a look? @dmohs , I approve your changes. Maybe if you approve mine, that's good enough?

**Review by panentheos — APPROVED** [review]
Looks good, tested locally.

**Pull request merged**